x-pack/plugins/security_solution/cypress/integration/cases/attach_alert_to_case.spec.ts

```diff
@@ -53,7 +53,8 @@ describe('Alerts timeline', () => {
       loadDetectionsPage(ROLES.platform_engineer);
     });
 
-    it('should allow a user with crud privileges to attach alerts to cases', () => {
+    // TODO: Passes locally, but fails on CI, need to investigate flake...
+    it.skip('should allow a user with crud privileges to attach alerts to cases', () => {
```

Comment:
Will investigate before merging, but [appears to be flake](https://buildkite.com/elastic/kibana-pull-request/builds/39905#f3d60a68-c517-45c5-b825-35f368d9a1f6) as it passes fine locally:
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/164557027-b53541d7-0a21-456f-8b3d-f2ce3374dfe6.png" />
</p>

Reply:
@MadameSheema -- found some flake here with opening the Alert context menu. Here's the change I made to wait till the icons are done loading and the button is enabled. This seemed to be the only signal other than click looping and waiting for the menu, but let me know if you have any thoughts here -- thanks! 🙂 

``` ts
export const expandFirstAlertActions = () => {
  cy.get(TIMELINE_CONTEXT_MENU_BTN).should('be.visible');
  cy.get(TIMELINE_CONTEXT_MENU_BTN).find('svg').should('have.class', 'euiIcon-isLoaded');
  cy.get(TIMELINE_CONTEXT_MENU_BTN).first().click({ force: true });
};
```
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/165845358-7da56171-ee9a-4517-932d-150cdee2602d.png" />
</p>